src/Infrastructure/Sulu/Content/SingleArticleSelectionContentType.php

```diff
@@ -32,7 +34,22 @@ public function getContentData(PropertyInterface $property)
             return null;
         }
 
-        return $this->articleRepository->getOneBy(['uuid' => $uuid]);
+        $dimensionAttributes = [
+            'locale' => $property->getStructure()->getLanguageCode(),
+            'stage' => DimensionContentInterface::STAGE_DRAFT,
+        ];
+
+        return $this->articleRepository->getOneBy(
```

Comment:
we need to resolve the content types this will just load raw article without resolved content types and excerpt.